docs/sources/operators-guide/monitor-grafana-mimir/dashboards/config/index.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: "Grafana Mimir Config dashboard"
-menuTitle: "Config"
-description: "View an example Config dashboard."
-weight: 50
 aliases:
-  - ../../visualizing-metrics/dashboards/config/
+  - /../../visualizing-metrics/dashboards/config/
+  - /docs/mimir/latest/operators-guide/monitoring-grafana-mimir/dashboards/config/
+description: View an example Config dashboard.
+menuTitle: Config
+title: Grafana Mimir Config dashboard
+weight: 50
 ---
 
 # Grafana Mimir Config dashboard
```

Comment:
```suggestion
# Grafana Mimir configuration dashboard
```

Reply:
Unrelated to this change and I believe the title matches the dashboard title